lib/internal/test_runner/runner.js

```diff
@@ -203,10 +199,15 @@ class FileTest extends Test {
     this.reportStarted();
```

Comment:
So is this calling a no-op function now?

Reply:
it is still called in other places:
https://github.com/nodejs/node/blob/806a516851f906c6cf0ca7b45dbcf499855b4c54/lib/internal/test_runner/test.js#L338
https://github.com/nodejs/node/blob/806a516851f906c6cf0ca7b45dbcf499855b4c54/lib/internal/test_runner/test.js#L692
https://github.com/nodejs/node/blob/806a516851f906c6cf0ca7b45dbcf499855b4c54/lib/internal/test_runner/test.js#L720

Reply:
I still don't love this, but I won't hold up this PR on it. 😄